gtsfm/two_view_estimator.py

```diff
@@ -398,10 +436,14 @@ def compute_relative_pose_metrics(
         Rotation error, in degrees
         Unit translation error, in degrees
     """
-    R_error_deg = comp_utils.compute_relative_rotation_angle(i2Ri1_computed, i2Ti1_expected.rotation())
-    U_error_deg = comp_utils.compute_relative_unit_translation_angle(
-        i2Ui1_computed, Unit3(i2Ti1_expected.translation())
-    )
+    if wTi1_expected is not None and wTi2_expected is not None:
+        i2Ti1_expected = wTi2_expected.between(wTi1_expected)
```

Comment:
cant the relative pose be passed in here, instead of computing it here? this function only needs relative pose, why ask for more?

Reply:
I have to compute it somewhere. I do not want to have this computation in the scene_optimizer class like we were doing before.

Reply:
okay